setup.cfg

```diff
@@ -21,3 +21,10 @@ exclude =
     */south_migrations/,
     */migrations/,
     */testsettings.py,
+
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = lizard_auth_client.testsettings
+norecursedirs = .venv data deploy design doc etc *.egg-info misc var
+python_files = tests.py
```

Comment:
Wat doet deze? Ziet eruit als "de tests staan alleen in dit bestand", maar pytest pikt ze automatisch wel op, toch?
Wel moet je 'em geloof ik `test_xyz.py` noemen met een underscore en werkt juist django's `tests.py` niet.